packages/sst/src/runtime/handlers/go.ts

```diff
@@ -72,7 +72,7 @@ export const useGoHandler = Context.memo(async () => {
               "go",
               "build",
               ...(ldFlags ? [`-ldflags "${ldFlags.join(" ")}"`] : []),
-              ...(buildTags ? [`-t ${buildTags.join(" ")}`] : []),
+              ...(buildTags ? [`-tags ${buildTags.join(" ")}`] : []),
```

Comment:
You're also defaulting buildTags to `[]`, so this is always going to be true. Maybe remove that default or check the length here?

Reply:
Thanks! That slipped through in the last PR.